### PR TITLE
[QUIC] Disable failing test on arm machines

### PR DESCRIPTION
**Pull request by ManickaP**
Contributes to https://github.com/dotnet/runtime/issues/82769

### CODE REVIEW

File: src/libraries/System.Net.Quic/tests/FunctionalTests/QuicListenerTests.cs [hunk 1]
```diff
@@ -203,6 +203,7 @@ public Task Listener_BacklogLimitRefusesConnection_ParallelClients_ClientThrows(
         [InlineData(100, 101)]
         [InlineData(15, 100)]
         [InlineData(10, 1_000)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/82769", typeof(PlatformDetection), nameof(PlatformDetection.IsArmOrArm64Process), nameof(PlatformDetection.IsInContainer))]
```
Comment (wfurt) [inline src/libraries/System.Net.Quic/tests/FunctionalTests/QuicListenerTests.cs:206]: Do we need the container part?

Comment (ManickaP) [reply src/libraries/System.Net.Quic/tests/FunctionalTests/QuicListenerTests.cs]: Removed.

**Review by wfurt — APPROVED** [review]
LGTM

**Pull request merged**